src/stores/RoleStore.js

```diff
@@ -55,6 +55,15 @@ class RoleStore extends DataStore {
     });
   }
 
+  /**
+   * The role with the highest position in the store
+   * @type {Role}
+   * @readonly
+   */
+  get highest() {
+    return this.reduce((prev, role) => !prev || role.comparePositionTo(prev) > 0 ? role : prev);
```

Comment:
Another option for that would be:
```javascript
return this.reduce((prev, role) => role.comparePositionTo(prev) > 0 ? role : prev, this.first());
```

Which sets the first role (normally the everyone role) as the first `prev` value. That'd skip one more check and improve performance and efficiency when checking for large `RoleStore`s. Remember that [`Collection#reduce`](https://discord.js.org/#/docs/main/master/class/Collection?scrollTo=reduce) is identical to [`Array#reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce), so you can take advantage of this.